src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FtpaAppellantPreparerTest.java

```diff
@@ -288,4 +318,54 @@ void should_not_allow_null_arguments() {
             .hasMessage("callback must not be null")
             .isExactlyInstanceOf(NullPointerException.class);
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {IN_FLIGHT_CASES_FTPA_OUT_OF_TIME_DATE, IN_FLIGHT_CASES_FTPA_IN_TIME_DATE_OOC})
+    void should_populate_ftpa_submission_out_of_time_to_correct_value_for_in_flight_cases_ooc(String input) {
```

Comment:
Better naming convention for variable input would be appealDate. But change only if you are pushing another commit, else its fine.